vial-gui

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+FILE=$(readlink "$0" || echo "$0")
+DIR=$(dirname $FILE)
```

Comment:
This can be done in a more POSIX compatible way with:
```shell
#!/usr/bin/env sh
DIR="$(CDPATH= cd -- "$(dirname -- "$0")" && pwd -P)"
```

While readlink tends to be available on GNU, Busybox, & BSD/OS X systems, it's non-standard & flags differ between them.

Reply:
good catch.